src/ch/interlis/ili2fme/Ili2Reader.java

```diff
@@ -1682,12 +1804,15 @@ private void mapAttributeDef(AttributeDef geomAttr, IFMEFeature ret, AttributeDe
 			attrNamePrefix="";
 		}
 		Type type=attr.getDomainResolvingAll();
-		if (type instanceof PolylineType 
-			|| type instanceof SurfaceOrAreaType
-			|| type instanceof CoordType){
-				if((type instanceof CoordType) && ((CoordType)type).getDimensions().length==1){
-					String numType = mapNumericType((NumericType)((CoordType)type).getDimensions()[0]);
-					ret.setSequencedAttribute(attrNamePrefix+attr.getName(),numType);
+		if (type instanceof LineType
+			|| type instanceof AbstractCoordType){
+				if((type instanceof AbstractCoordType) && ((AbstractCoordType)type).getDimensions().length==1){
+					String numType = mapNumericType((NumericType)((AbstractCoordType)type).getDimensions()[0]);
+					if(type instanceof MultiCoordType){
+						ret.setSequencedAttribute(attrNamePrefix+attr.getName()+"{}",numType);

```

Comment:
das "{}" ist zuviel

Reply:
@claeis In diesem Kontext werden Koordinaten mit nur einer Dimension auf einen einfachen Numerischen Typ gemapped. Für MultiCoord können hier dann mehrere Numerische Werte resultieren. Es ist also nach meinem Verständnis korrekt und funktioniert auch in FME.